sequencer/src/lib.rs

```diff
@@ -93,13 +95,20 @@ pub mod network {
     }
 
     #[derive(Clone, Copy, Default)]
-    pub struct Combined;
+    pub struct Networks;
```

Comment:
I think this name is too vague, what about like `Production` to distinguish it from `Memory` which is used only in tests?